<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.528, y1:1, x2:0.511, y2:0, stop:0 rgba(125, 125, 125, 255), stop:0.471591 rgba(219, 219, 219, 255))</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" rowspan="4">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QVTKWidget" name="qvtkWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1054</width>
          <height>772</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="rgb_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>480</width>
          <height>480</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="depth_label">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>120</y>
          <width>480</width>
          <height>480</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">border-style:none;
border:1px solid #242424;
color:#DCDCDC;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);</string>
      </property>
      <property name="text">
       <string>上傳</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidgetFilelist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-style:none;
border:1px solid #242424;
color:#DCDCDC;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);</string>
          </property>
          <property name="text">
           <string>切割</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-style:none;
border:1px solid #242424;
color:#DCDCDC;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);</string>
          </property>
          <property name="text">
           <string>參數計算</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidgetParameter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size : 15px;
background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 231, 231);</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>編輯</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>顏色</string>
     </property>
     <addaction name="action_setColor"/>
     <addaction name="action_heightRamp"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_pickPoints"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Top">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 170);</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_cut"/>
   <addaction name="action_setColor"/>
   <addaction name="action_heightRamp"/>
   <addaction name="action_pickPoints"/>
   <addaction name="action_getimage"/>
   <addaction name="action_play"/>
   <addaction name="action_delete"/>
   <addaction name="action_update"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Left">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 170)</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_reset"/>
   <addaction name="action_up"/>
   <addaction name="action_front"/>
   <addaction name="action_left"/>
   <addaction name="action_back"/>
   <addaction name="action_right"/>
   <addaction name="action_bottom"/>
   <addaction name="action_frontIso"/>
   <addaction name="action_backIso"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_Right">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 170)</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionaction_delete_2"/>
   <addaction name="actionaction_paint"/>
   <addaction name="actionaction_export"/>
   <addaction name="actionaction_delete_3"/>
   <addaction name="actionaction_rename"/>
   <addaction name="actionaction_UIcolorChange"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 170)</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_up">
   <property name="icon">
    <iconset>
     <normaloff>image/up.png</normaloff>image/up.png</iconset>
   </property>
   <property name="text">
    <string>俯視圖</string>
   </property>
  </action>
  <action name="action_front">
   <property name="icon">
    <iconset>
     <normaloff>image/front.png</normaloff>image/front.png</iconset>
   </property>
   <property name="text">
    <string>前視圖</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_left">
   <property name="icon">
    <iconset>
     <normaloff>image/left.png</normaloff>image/left.png</iconset>
   </property>
   <property name="text">
    <string>左視圖</string>
   </property>
   <property name="toolTip">
    <string>左視圖</string>
   </property>
  </action>
  <action name="action_back">
   <property name="icon">
    <iconset>
     <normaloff>image/back.png</normaloff>image/back.png</iconset>
   </property>
   <property name="text">
    <string>後視圖</string>
   </property>
  </action>
  <action name="action_right">
   <property name="icon">
    <iconset>
     <normaloff>image/right.png</normaloff>image/right.png</iconset>
   </property>
   <property name="text">
    <string>右視圖</string>
   </property>
  </action>
  <action name="action_bottom">
   <property name="icon">
    <iconset>
     <normaloff>image/bottom.png</normaloff>image/bottom.png</iconset>
   </property>
   <property name="text">
    <string>底視圖</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>image/open.png</normaloff>image/open.png</iconset>
   </property>
   <property name="text">
    <string>打開</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_heightRamp">
   <property name="icon">
    <iconset>
     <normaloff>image/heightRamp.png</normaloff>image/heightRamp.png</iconset>
   </property>
   <property name="text">
    <string>高度渲染</string>
   </property>
  </action>
  <action name="action_pickPoints">
   <property name="icon">
    <iconset>
     <normaloff>image/pickPoints.png</normaloff>image/pickPoints.png</iconset>
   </property>
   <property name="text">
    <string>選點</string>
   </property>
  </action>
  <action name="action_frontIso">
   <property name="icon">
    <iconset>
     <normaloff>image/frontIso.png</normaloff>image/frontIso.png</iconset>
   </property>
   <property name="text">
    <string>前軸測</string>
   </property>
  </action>
  <action name="action_backIso">
   <property name="icon">
    <iconset>
     <normaloff>image/backIso.png</normaloff>image/backIso.png</iconset>
   </property>
   <property name="text">
    <string>後軸測</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="icon">
    <iconset>
     <normaloff>image/reset.png</normaloff>image/reset.png</iconset>
   </property>
   <property name="text">
    <string>重設</string>
   </property>
  </action>
  <action name="action_setColor">
   <property name="icon">
    <iconset>
     <normaloff>image/color.png</normaloff>image/color.png</iconset>
   </property>
   <property name="text">
    <string>點雲顏色</string>
   </property>
  </action>
  <action name="action_getimage">
   <property name="icon">
    <iconset>
     <normaloff>image/camera.png</normaloff>image/camera.png</iconset>
   </property>
   <property name="text">
    <string>取得影像</string>
   </property>
  </action>
  <action name="action_play">
   <property name="icon">
    <iconset>
     <normaloff>image/play.png</normaloff>
     <normalon>image/play.png</normalon>image/play.png</iconset>
   </property>
   <property name="text">
    <string>action_play</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset>
     <normaloff>image/pause1.png</normaloff>image/pause1.png</iconset>
   </property>
   <property name="text">
    <string>action_stop</string>
   </property>
  </action>
  <action name="actionaction_2d">
   <property name="icon">
    <iconset>
     <normaloff>image/2d.svg</normaloff>image/2d.svg</iconset>
   </property>
   <property name="text">
    <string>action_2d</string>
   </property>
  </action>
  <action name="actionaction_3d">
   <property name="icon">
    <iconset>
     <normaloff>image/camera.png</normaloff>image/camera.png</iconset>
   </property>
   <property name="text">
    <string>action_3d</string>
   </property>
  </action>
  <action name="action_update">
   <property name="icon">
    <iconset>
     <normaloff>image/virus_update.png</normaloff>image/virus_update.png</iconset>
   </property>
   <property name="text">
    <string>action_update</string>
   </property>
  </action>
  <action name="actionaction_delete_2">
   <property name="icon">
    <iconset>
     <normaloff>image/NonMode.png</normaloff>image/NonMode.png</iconset>
   </property>
   <property name="text">
    <string>action_non</string>
   </property>
  </action>
  <action name="actionaction_paint">
   <property name="icon">
    <iconset>
     <normaloff>image/paint.png</normaloff>image/paint.png</iconset>
   </property>
   <property name="text">
    <string>action_paint</string>
   </property>
  </action>
  <action name="actionaction_export">
   <property name="icon">
    <iconset>
     <normaloff>image/export.png</normaloff>image/export.png</iconset>
   </property>
   <property name="text">
    <string>action_export</string>
   </property>
  </action>
  <action name="actionaction_delete_3">
   <property name="icon">
    <iconset>
     <normaloff>image/recycle_bin.png</normaloff>image/recycle_bin.png</iconset>
   </property>
   <property name="text">
    <string>action_delete</string>
   </property>
  </action>
  <action name="actionaction_rename">
   <property name="icon">
    <iconset>
     <normaloff>image/rename.png</normaloff>image/rename.png</iconset>
   </property>
   <property name="text">
    <string>action_rename</string>
   </property>
  </action>
  <action name="actionaction_UIcolorChange">
   <property name="icon">
    <iconset>
     <normaloff>image/UI_ColorChange.png</normaloff>image/UI_ColorChange.png</iconset>
   </property>
   <property name="text">
    <string>action_UIcolorChange</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>image/save1.png</normaloff>image/save1.png</iconset>
   </property>
   <property name="text">
    <string>action_save</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset>
     <normaloff>image/cut.png</normaloff>image/cut.png</iconset>
   </property>
   <property name="text">
    <string>action_cut</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pclviewer.qrc"/>
 </resources>
 <connections/>
</ui>
